test-data/unit/semanal-typeddict.test

```diff
@@ -0,0 +1,16 @@
+-- Semantic analysis of typed dicts
+
+[case testCanDefineTypedDictType]
+from mypy.typing import TypedDict
+Point = TypedDict('Point', {'x': int, 'y': int})
+[out]
```

Comment:
I think you'll need to add `test-data/unit/lib-stub/mypy/__init__.pyi` and `test-data/unit/lib-stub/mypy/typing.pyi`, as otherwise mypy won't find the `mypy.typing` module. Test cases look for stubs here and they don't use typeshed stubs. You can write a minimal stub that lets the tests pass.

Also, the default builtins stubs used in test cases don't include `dict`. You should be able to add this line to the test case for `dict` to work (just before `[out]`):

```
[builtins fixtures/dict.pyi]
```